content/11.raw-data-analysis.md

```diff
@@ -1,9 +1,54 @@
+
+In typical bottom-up proteomics experiments, proteins are digested into peptides and further analyzed with LC-MS/MS systems. 
+Peptides can have different PTMs and ionize differently depending on their length and amino acid distributions. 
+Therefore, mass spectrometers often record different charge and modification states of one single peptide. 
+The entity that is recorded on a mass spectrometer is usually referred to as a precursor ion (peptide with its modification and charge state). 
+This precursor ion is fragmented and the precursor or peptide sequences are obtained though spectral matching. 
+The quantity of a precursor is estimated with various methods.
+The measured precursor quantities are combined to generate a peptide quantity. 
+Peptides are also often combined into a protein group through protein inference, which combines multiple peptide identifications into a single protein identification [@DOI:https://doi.org/10.1074/mcp.R500012-MCP200] [@DOI:10.1016/j.jprot.2016.08.002].
+Protein inference is still a challenge in bottom-up proteomics. 
+
+Due to the inherent differences in the data structures of DDA and DIA measurements, there exist different types of software that can facilitate the steps mentioned above. 
+The existing software for DDA and DIA analysis can be further divided into freeware and non-freeware:
+
+<!-- TODO: should we make a table for this part? with primary citation and link to the website -->
+
+DDA freeware: 
+- MaxQuant [@URL:https://www.maxquant.org/]
+- MSFragger [@URL:https://msfragger.nesvilab.org/] 
+- Mascot (for smaller data sets) [@URL:https://www.matrixscience.com/] [@PMID:10612281]
+- MS-GF+ [@DOI:https://doi.org/10.1038/ncomms6277]
+
+DIA freeware:
+- MaxDIA (within MaxQuant) [@URL:https://www.maxquant.org/]
+- Skyline [@URL:https://skyline.ms/project/home/software/Skyline/begin.view]
+- DIA-NN [@DOI:https://doi.org/10.1038/s41592-019-0638-x]
+Targeted proteomics freeware:
+- Skyline [@URL:https://skyline.ms/project/home/software/Skyline/begin.view]
+
+DDA non-freeware:
+- ProteomeDiscoverer[@URL:https://www.thermofisher.com/ch/en/home/industrial/mass-spectrometry/liquid-chromatography-mass-spectrometry-lc-ms/lc-ms-software/multi-omics-data-analysis/proteome-discoverer-software.html]
+- Mascot (for larger data sets) [@URL:https://www.matrixscience.com/]
+- Spectromine [@URL:https://biognosys.com/software/spectromine/?gclid=Cj0KCQiAoY-PBhCNARIsABcz770mjUz6iavBr9Ql7RPUdMvaHu9RYgPNrEfZco1wExEeoFwnQXuCHscaAlgBEALw_wcB]
+- PEAKS [@URL:https://www.bioinfor.com/peaks-studio/]
+
+DIA non-freeware:
+- Spectronaut [@URL:https://biognosys.com/software/spectronaut/?gclid=Cj0KCQiAoY-PBhCNARIsABcz770nuaU2SgIriS-ZJJGsC6CtzXc9AC8b9K3w5FlFDsDfGtnuUjIhankaAvegEALw_wcB]
```

Comment:
URL citations that contain special characters like `?` cause problems when getting the reference metadata.  You can fix that by wrapping everything in curly braces `{ }` like
`@{https://biognosys.com/software/spectronaut/?gclid=Cj0KCQiAoY-PBhCNARIsABcz770nuaU2SgIriS-ZJJGsC6CtzXc9AC8b9K3w5FlFDsDfGtnuUjIhankaAvegEALw_wcB}`

Note that the `url:` prefix is optional  `https:` can also be used directly when citing a URL.

Reply:
Thanks a lot for the feedback that's really helpful! I will fix this later today.